assets/css/footer.css

```diff
@@ -0,0 +1,29 @@
+footer{
+    background:#1e1e1e;
+    height:40vh;
+    width:100%;
+    box-sizing:border-box;
+    padding:50px 0;
+    color:#fff;
+}
+.footer-heading{
+    display:flex;
+    justify-content: center;
+}
+.footer-heading span{
+    display:block;
+    text-align:center;
+}
+.footer-item{
+    height:80%;
+    display: flex;
+    justify-content:center;
+    align-items: flex-end;
+}
+.footer-item a{
+    color:#adcce9;
+}
+.footer-item a:hover{
+    color:#5e5e5e;
+    text-decoration:none;
```

Comment:
Wydaje mi się że to powinno być zdedfiniowane w .footer-item a

Nie jestem pewien - nie mam teraz dostępu do projektu graficznego. Sprawdź dokładnie.

Reply:
Tak sie może wydawać ale po najechaniu kolor się zmienia jeszcze na inny.